### PR TITLE
Change "referrer" to "referer" in app controller.

### LINKED ISSUE #117: Fix "Rubocop/Rails/RequestReferer" issue in app/controllers/application_controller.rb
Use `request.referer` instead of `request.referrer`.

https://codeclimate.com/github/ChaelCodes/HuntersKeepers/app/controllers/application_controller.rb#issue_5eaed482fcf8990001000103

### PR DESCRIPTION
**Pull request by LeftoverCurry**
Closes #119

### CODE REVIEW

**Comment by LeftoverCurry** [comment]
@ChaelCodes While this does close the issue at hand, it seems to expose another.  The functionality of the redirect in case of an unauthorized user isn't currently covered under the test suite, and I do feel like it's something that should be tested.  I looked through the suite, but since there is no application controller spec (and no user accessible methods within the controller itself) I was unsure of how (or if, for that matter) you would want to structure any testing.

**Comment by ChaelCodes** [comment]
Hey @LeftoverCurry , turns out referrer is the right spelling. Can we change this PR to be a rubocop config change?

**Comment by LeftoverCurry** [comment]
@ChaelCodes Got it switched out.

File: .rubocop.yml [hunk 1]
```diff
@@ -8,3 +8,6 @@ AllCops:
 
 Layout/EmptyLineAfterGuardClause:
   Enabled: false
+
+Rails/RequestReferer:
+  EnforcedStyle: referrer
```
Comment (ChaelCodes) [inline .rubocop.yml:13]:
```suggestion
  EnforcedStyle: referrer
  
```
Uh oh! Needs a new line.

Comment (LeftoverCurry) [reply .rubocop.yml]: Got it, homie!

**Pull request merged**